src/v3/src/types/schema.ts

```diff
@@ -134,6 +135,11 @@ export interface UISchemaElement {
    * view/step within the {@link StepperLayout} this element belongs to.
    */
   viewIndex?: number;
+  /**
+   * This function should be defined when your text contains <$1> tags
+   * which require custom interpolation
+   */
+  contentTransformer?: Options['transform'];
```

Comment:
Is there any possibility this would be anything other than the function that `util/getHTMLTransformer` returns?

Reply:
Oh I didn't think about the fact that we need to pass parameters to this HOF to get the transformer we want. I think this makes more sense now with that in mind.

Reply:
It shouldn't be, there are other options the ReactHTMLParser function accepts but I had no need to use those. If we want to expose all possible options to our transformers we could convert that to something like `parserOptions` and have our transformer fill in the object as needed. I went with just the transform function to limit the capabilities but I see no reason against exposing the entire `Options` object to our transformers. Curious what you think about that.